ament_cppcheck/ament_cppcheck/main.py

```diff
@@ -86,6 +91,9 @@ def main(argv=sys.argv[1:]):
            '--xml-version=2']
     if args.language:
         cmd.extend(['--language={0}'.format(args.language)])
+    if len(args.include_dirs) > 0:
```

Comment:
```suggestion
    if args.include_dirs:
```

An empty argument will default to None for which len(None) is undefined. 

```
>>> parser=argparse.ArgumentParser()
>>> parser.add_argument('--asdf', nargs='*')
_StoreAction(option_strings=['--asdf'], dest='asdf', nargs='*', const=None, default=None, type=None, choices=None, help=None, metavar=None)
>>> args = parser.parse_args()
>>> args
Namespace(asdf=None)
>>> len(args.asdf)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: object of type 'NoneType' has no len()

```

Reply:
Thanks! Nice catch